UEBPrep/app/views/cards/_image_card_content.erb

```diff
@@ -1,14 +1,14 @@
 <div class="row user-card-height">
   <div class="user-card panel app-block app-roundedbox small-12 small-centered columns">
     <div class="sign">
-      <h1><%= content.title %></h1>
+      <h1><%= @card.nil? ? content.title : @card.content.title %></h1>
```

Comment:
we have two actions, show and edit using this same partial, with conflicting variables. Will need to resolve this some day.